content_schemas/formats/link_collection.jsonnet

```diff
@@ -0,0 +1,35 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "link_collection",
+  base_path: "optional",
+  routes: "optional",
+  rendering_app: "optional",
+  title: "optional",
```

Comment:
I wonder if we should make this mandatory...

Just thinking if we _do_ ever end up using `link_collection` for more than one content item, it might get hard to tell them apart if we don't strongly encourage the publishing apps to identify them somehow...

I guess also we could use the title for the "Popular on GOV.UK" heading on the homepage... If we ended up using the popular links in other places, would we want to use the same title everywhere, and be able to update it once? Maybe?

Reply:
Yeah, that makes sense again, title kind of becomes the identifier. thank you.